### DISCUSSION TITLE
Improve UX for removing a tag

### DISCUSSION THREAD

**Discussion by thoughtgap** [LeoKlaus/Paperparrot#188] [Feature Requests]
I'm working with a `new` tag as inbox indicator. Which means I need to remove this tag from all inbox documents. Currently this is cumbersome:

1. Tap the Tags
2. Scroll through my very long list (115 tags), past the Capital letters, until I make it to the lower case n. Tap the tag again.
3. It gets unassigned

I'm proposing adding a tag display like this `(X) new` - tapping the `X` removes the tag. Similar to the web version of paperless: 
![image](https://github.com/LeoKlaus/Paperparrot/assets/3226639/3f701bf2-954d-4624-b780-63755f010376)

Another option could be to collect assigned tags at the top of the lists, so the scrolling part is not required.

**Comment by LeoKlaus** [comment]
I've added new menus for all attributes with the latest TestFlight release. If you want to join the TestFlight beta, you can do so via the button in the settings tab in Paperparrot (right above "help" on the bottom).

Additionally, I plan to mirror the new Paperless feature that lets you automatically remove inbox tags when saving.
